apimonitor/views.py

```diff
@@ -118,6 +121,72 @@ def create(self, request, *args, **kwargs):
     def get_queryset(self):
         queryset = APIMonitor.objects.filter(user=self.request.user)
         return queryset
+
+    def retrieve(self, request, pk):          
+
+        queryset = self.filter_queryset(self.get_queryset())
```

Comment:
```suggestion
    def retrieve(self, request, pk=None):          
        queryset = self.filter_queryset(self.get_queryset())
```

nitip diubah pk=None ferr biar sesuai sama docsnya wkkwk
https://www.django-rest-framework.org/api-guide/viewsets/